smilerating/src/main/java/com/hsalf/smilerating/SmileRating.java

```diff
@@ -36,6 +35,13 @@
     private int mAngryColor = Color.parseColor("#f29a68");
     private int mNormalColor = Color.parseColor("#f2dd68");
```

Comment:
There is no need for mAngryColor or even mNormalColor when you add colors for each of the smiley faces.